config/config.yaml

```diff
@@ -0,0 +1,330 @@
+      trigger-on-minimum-admin-area-in-drought-extent: 0.5 
+    Climate_Region:
+      - name: National
+        climate-region-code: 1
+        leadtime:
+          Jan: 
+            - MAM: "2-month"
+          Feb: 
+            - MAM: "1-month"
+          Mar: 
+            - MAM: "0-month"
+          Apr: 
+            - MAM: "0-month"
+          May: 
+            - MAM: "0-month"
+          Jun: 
+            - OND: "3-month"
+          Jul: 
+            - OND: "2-month"
+          Aug: 
+            - OND: "1-month"
+          Sep: 
+            - OND: "0-month"
+          Oct: 
+            - OND: "0-month"
+          Nov: 
+            - OND: "0-month"
+          Dec: 
+            - MAM: "3-month"
+  - name: LSO
+    admin-levels: 
+      - 1
+    trigger-on-lead-time: 3 # number of months to trigger alert
+    pipeline-will-trigger-portal: disable
+    classify-alert-on: disable
+    alert-on-minimum-probability:
+      min: 0.65
+      med: 0.75
+      max: 0.85
+    trigger_model: 
+      model: seasonal_rainfall_forecast
+      trigger-on-minimum-probability: 0.7 # based on ensamble member probability compared against lower tercile 
+      trigger-on-minimum-probability-drought-extent: 0.6  # for drought extent using below average sesonal rain as a proxy
+      trigger-on-minimum-admin-area-in-drought-extent: 0.5 #for drought extent(grid) overlap with  admin polygons 
+    Climate_Region:
+      - name: National
+        climate-region-code: 1
+        leadtime:
+          Jan: 
+            - WET_SEASON: "0-month"
+          Feb: 
+            - WET_SEASON: "0-month"
+          Mar: 
+            - DRY_SEASON: "0-month"
+          Apr: 
+            - DRY_SEASON: "0-month"
+          May: 
+            - DRY_SEASON: "0-month"
+          Jun: 
+            - DRY_SEASON: "0-month"
+          Jul: 
+            - WET_SEASON : "3-month"
+          Aug: 
+            - WET_SEASON: "2-month"
+          Sep: 
+            - WET_SEASON: "1-month"
+          Oct: 
+            - WET_SEASON: "0-month"
+          Nov: 
+            - WET_SEASON: "0-month"
+          Dec: 
+            - WET_SEASON: "0-month"
+  - name: UGA
+    admin-levels:
+      - 2
+    trigger-on-lead-time: 3 # number of months to trigger alert
+    pipeline-will-trigger-portal: disable
+    classify-alert-on: disable
+    alert-on-minimum-probability:
+      min: 0.65
+      med: 0.75
+      max: 0.85
+    trigger_model: 
+      model: seasonal_rainfall_forecast
+      trigger-on-minimum-probability: 0.6 # based on ensamble member probability compared against lower tercile 
+      trigger-on-minimum-probability-drought-extent: 0.6  # for drought extent using below average sesonal rain as a proxy
+      trigger-on-minimum-admin-area-in-drought-extent: 0.5 #for drought extent(grid) overlap with  admin polygons 
+    Climate_Region:
+      - name: National
+        climate-region-code: 1
+        leadtime:
+          Jan: 
+            - Long_Rains: "2-month"
```

Comment:
Maybe we can leave this for now, as Uganda pipeline will not be implemented yet. But when we almost implemented this years ago we had Karamoja region separate with 1 season April-Oct, and the rest of the country having 2 season (MAM + OND). This is how it's still configured in IBF repo and e.g. IBF-demo. Will that change? Either way, fine to keep as is for now right?

![image](https://github.com/user-attachments/assets/7b18774e-26ef-4f38-a53f-5f2575cb1519)